deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/__init__.py

```diff
@@ -31,6 +31,7 @@
 CUSTOM_ARCHLINUX_JSON_FEED = 'custom_archlinux_feed.json'
 CUSTOM_ALAS_JSON_FEED = 'custom_alas_feed.json'
 CUSTOM_ALAS2_JSON_FEED = 'custom_alas2_feed.json'
+CUSTOM_JAMMY_OVAL_FEED = 'custom_ubuntu22_oval_feed.xml'
```

Comment:
Analyzing the behavior of Canonical _18_ or _20_, we can see that for `Ubuntu 22` the procedure does not change, so we would only need to add the `Ubuntu 22 version` to the canonical feed and use this feed. So you should remove this `CUSTOM_JAMMY_OVAL_FEED = 'custom_ubuntu22_oval_feed.xml'` line and modify it to use  `CUSTOM_CANONICAL_OVAL_FEED = 'custom_canonical_oval_feed.xml'`